conan/tools/files/patches.py

```diff
@@ -39,7 +39,7 @@ def patch(conanfile, base_path=None, patch_file=None, patch_string=None, strip=0
     :param kwargs: Extra parameters that can be added and will contribute to output information
     """
 
-    patch_type = kwargs.get('patch_type')
+    patch_type = kwargs.get('patch_type') or ("file" if patch_file else "string")
```

Comment:
It seems this could be breaking the tests, other types like "backport" are being ovewrritten?
https://ci.conan.io/blue/organizations/jenkins/ConanTestSuitev2/detail/PR-14177/1/pipeline/9

Reply:
It should respect any non-empty value of `patch_type` that gets passed in. I'm not familiar with the test output conventions, but isn't it complaining about the new/modified "Apply patch (file)" lines?

Remember, the old behavior was that `patch_file` without (`patch_type` or `patch_description`) printed nothing, and `patch_type` could be left unset and e.g. `"(file)"` would not be printed.